x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/importing.md

```diff
@@ -0,0 +1,72 @@
+Feature: Importing Prebuilt Rules
+
+  Scenario: Importing an unmodified prebuilt rule with a matching rule_id and version
+    Given the import payload contains a prebuilt rule with a matching rule_id and version, identical to the published rule
+    When the user imports the rule
+    Then the rule should be created or updated
```

Comment:
Is it worth having the specificity between these two here? What causes the rule to be created vs updated?

Reply:
@dplumlee @pborgonovi brought up similar questions, which were discussed [here](https://github.com/elastic/kibana/pull/191116#discussion_r1813447141), but the TL;DR: 

> When the test states Then the rule should be created or updated, that's really a shorthand for:

> If the rule is new, create it
> If the rule exists, update or reject based on the value of overwrite

But since that's all orthogonal to the prebuilt customization stuff, I didn't think it was an important thing to split out in these tests.